src/lib/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface Product {
```

Comment:
the file says types and we are using interfaces?

Reply:
but we are using this interface to give type to Product. what should I do here . can I change file name to interfaces.